index.html

```diff
@@ -2,22 +2,36 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
-    <title>Type your word!</title>
+    <title>Play</title>
+    <link href='https://fonts.googleapis.com/css?family=Raleway:400,700' rel='stylesheet' type='text/css'>
+    <script
+    src="js/jquery-3.2.1.js"></script>
+    <script type="text/javascript" src="js/main.js"></script>
     <script type="text/javascript" src="js/1.js"></script>
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <div class="container">
-      <h1 align="center">TYPE YOUR WORDS!</h1>
-      <div class="buttons">
-        <a href="pages/1.html">
-          <button id="play" type="button" name="button">Play</button></a>
-          <button type="button" name="button">Options</button>
-      </div>
-      <div class="leaderboard">
-        <a href="pages/leaderboard.html">
-          <img src="images/trophy.png" alt=""></a>
+  <div class="container">
+		<h1></h1>
+		<div class="outerbox">
+			<div class="scorebox">
+				<p class="disp">Score</p>
+        <p class="playerScore">0</p>
```

Comment:
Please check the indentation here please

Reply:
This is the problem I was talking about. The indentation that appears on here isn't the correct one...and I don't know what's causing it.